File: js/ejer22.js
lado1=5
lado2=10
lado3=15

 //Determinar el tipo de triángulo
  if (lado1 === lado2 && lado2 === lado3) {
    console.log("El triángulo Equilátero");
} else if (lado1 === lado2 || lado1 === lado3 || lado2 === lado3) {
         console.log("El triángulo Isósceles");
 } else {
     console.log("El triángulo Escaleno");
                }

File: python/ejer22.py
lado1=5
lado2=5
lado3=10

#determinar el tipo de triangulo

if lado1==lado2 and lado2==lado3:
    print("tríangulo equilátero")

elif lado1==lado2 | lado1==lado3 | lado2==lado3:
    print("tríangulo isósceles")

else:
    print("tríangulo escaleno")
